django/db/backends/base/schema.py

```diff
@@ -1534,6 +1535,9 @@ def _field_should_be_altered(self, old_field, new_field, ignore=None):
             new_field.column
         ) or (old_path, old_args, old_kwargs) != (new_path, new_args, new_kwargs)
 
+    def _fk_constraints_should_be_recreated(self):
+        return self.connection.vendor == "mysql"
```

Comment:
Just FYI Simon asked that a feature flag be created so that we don't need to check specific vendors. Do know how to add a flag? If you're at DjangoCon sprints then someone may be able to easily you through it ,👍

Reply:
Thanks @shangxiao I will look into this!